virttest/qemu_devices/qcontainer.py

```diff
@@ -3091,3 +3093,82 @@ def _emulator_setup(binary, extra_options=None):
         devices.append(tpm_model_dev)
 
         return devices
+
+    def numa_hmat_lb_define_by_params(self, nodeid, initiator, params):
+        """
+        Create numa hmat-lb device by params
```

Comment:
Technically, neither numa nor hmat-lb is a device. It's used to describe the attributes between numa nodes.

Reply:
> Technically, neither numa nor hmat-lb is a device. It's used to describe the attributes between numa nodes.

@huangyum Would it be better if replace it with the world `object`?

Reply:
I guess we could just remove 'device'.